test/load-tests/env/start-server.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+docker cp ./test/load-tests/env/dump.tar.gz openhim-mongo:/ &&
+docker exec -it openhim-mongo bash -c "tar -xzvf dump.tar.gz dump ; mongorestore /dump" &&
```

Comment:
We dont seem to do anything with the mongo container, so the data will not exist when running the load tests